indigo_api/views/attachments.py

```diff
@@ -51,6 +68,13 @@ def view(self, request, *args, **kwargs):
         attachment = self.get_object()
         return view_attachment(attachment)
 
+    @action(detail=True, methods=['GET'])
+    def as_pdf(self, request, *args, **kwargs):
+        attachment = self.get_object()
+        if attachment.mime_type in DOCX_MIME_TYPES:
```

Comment:
`soffice` is pretty good at converting almost anything to a PDF. So perhaps rather check if it's NOT a pdf, and then try to convert it.